lib/cocoapods/target/pod_target.rb

```diff
@@ -61,7 +68,7 @@ def initialize(specs, target_definitions, sandbox, scoped = false)
     #
     def scoped
       target_definitions.map do |target_definition|
-        PodTarget.new(specs, [target_definition], sandbox, true).tap do |target|
+        PodTarget.new(specs, [target_definition], sandbox, target_definition.label).tap do |target|
```

Comment:
ok, so the target definition label is always _a_ correct scope suffix, but it is the most broad and therefore undesirable? cool :+1: